core/src/confidence.rs

```diff
@@ -1,14 +1,21 @@
-use crate::result::{Error, Result};
-use crate::service::Service;
+use crate::{
+    result::{Error, Result},
+    service::Service,
+};
 use solana_runtime::bank::Bank;
-use solana_vote_api::vote_state::VoteState;
-use solana_vote_api::vote_state::MAX_LOCKOUT_HISTORY;
-use std::collections::HashMap;
-use std::sync::atomic::{AtomicBool, Ordering};
-use std::sync::mpsc::{channel, Receiver, RecvTimeoutError, Sender};
-use std::sync::{Arc, RwLock};
-use std::thread::{self, Builder, JoinHandle};
-use std::time::Duration;
+use solana_vote_api::vote_state::{VoteState, MAX_LOCKOUT_HISTORY};
+use std::{
+    collections::HashMap,
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        mpsc::{channel, Receiver, RecvTimeoutError, Sender},
+        Arc, RwLock,
+    },
+    thread::{self, Builder, JoinHandle},
+    time::Duration,
+};
+
+pub const DEFAULT_PERCENT_CONFIDENCE: f64 = 0.66667;
```

Comment:
(Well, naturally I wanted to suggest `(2_f64 / 3_f64).next_after(std::f64::INFINITY)`, but it seems that rust unfortunately deprecated this: https://doc.rust-lang.org/1.14.0/std/primitive.f32.html#method.next_after)